spec/models/conference_spec.rb

```diff
@@ -778,6 +778,35 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+
+    before(:each) do
+      @conference = create(:conference)
+
+      selected_schedule = create(:schedule, program: @conference.program)
```

Comment:
creation of variable should be in a `let`, lazy evaluation and you always know where to fine all the variables that you use in your test.